src/sentry/processing/backpressure/monitor.py

```diff
@@ -118,7 +118,10 @@ def start_service_monitoring() -> None:
             time.sleep(options.get("backpressure.monitoring.interval"))
             continue
 
-        with sentry_sdk.start_transaction(name="backpressure.monitoring", sampled=True):
+        with sentry_sdk.start_transaction(
+            name="backpressure.monitoring",
+            custom_sampling_context={"sample_rate": 1.0},
```

Comment:
For now you can also leave `sampled=True` here. I'd change this only once we get rid of `sampled`.